server/server.js

```diff
@@ -169,6 +169,23 @@ app.get('/box_category',(req, res) =>{
   });
 });
 
+
+/* ------------------------ category_content------------------------*/
+
+app.get('/category/:name/category_content/:type',(req, res) =>{
```

Comment:
contents of a category filtered by a type = /categories/:categoryId/contents?type=...